bower.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "stencil-select",
+  "version": "0.1.0",
+  "description": "A styleable, accessible <select> component.",
+  "authors": [
+    "Mobify"
+  ],
+  "keywords": [
+    "mobify",
+    "components",
+    "patterns"
+  ],
+  "main": "",
+  "dependencies": {
+    "spline": "1.0.1"
+  },
+  "devDependencies": {
+    "vellum": "~1.0.0",
+    "modularized-normalize-scss": "~3.0.1"
+  },
```

Comment:
BTW only depending on a shared variables file is totally a valid way to go here, IMO.